parsing/parchmint-parser.js

```diff
@@ -243,7 +242,7 @@ class ParchmintParser {
                 this.layers.push(this.getParsedLayer(value));
             } else {
                 this.valid = false;
-                console.log('Parser: Duplicate IDs (' + value['id'] + ') found in the "layers" key. Skipping Layer' +
+                console.log('Parser: Duplicate IDs (' + value['id'] + ') found in the layers array. Skipping Layer' +
                         ' with name "' + value['name'] + '" at index ' + index + '.');
```

Comment:
You might want to either not do this or do it in a different PR, but you can also use template literals here

```js
console.log(`Parser: Duplicate IDs (${value['id']}) found in the layers array. Skipping Layer with name "${value['name']}" at index ${index}`);
```

Reply:
They will be changed by #66.